### PR TITLE
Feat/xd 104 - Case CRUD Operations

### PR DESCRIPTION
**Pull request by Sabo2k**

### CODE REVIEW

File: libs/cases/frontend/view/src/lib/components/create-case/create-case.component.ts [hunk 1]
```diff
@@ -1,70 +1,129 @@
...
+        return this.createCaseForm.phone;
+    }
+
+    public getEmailValue() {
+        return this.createCaseForm.email;
+    }
+
+    public getTextValue() {
+        return this.createCaseForm.text;
+    }
```
Comment (KonsumGandalf) [inline libs/cases/frontend/view/src/lib/components/create-case/create-case.component.ts]: 1. If you want to use encapsulation to set the values via template driven approach and *not* directly use *dynamicForms* and use encapsulation. You need to use `get `and `set `here as keyword: https://www.typescripttutorial.net/typescript-tutorial/typescript-getters-setters/

How do use dynamic forms (in my perception better with so many validators)
https://angular.dev/guide/forms/dynamic-forms

2. Some quotes are in `"` Reformat.

File: libs/cases/frontend/view/src/lib/components/create-case/create-case.component.ts [hunk 2]
```diff
@@ -1,70 +1,129 @@
...
+            title: formData.title,
+            type: formData.selectType,
+            status: CaseStatus.OPEN,
+            description: formData.text,
+            source: 'Internal System A',
+            priority: formData.selectPriority,
+            createdBy: formData.email,
+            eTag: 'etag_value_here'
+        };
+    }
```
Comment (KonsumGandalf) [inline libs/cases/frontend/view/src/lib/components/create-case/create-case.component.ts]: Why dont you create interfaces for this?

File: libs/cases/frontend/view/src/lib/components/create-case/date-dropdown-accessor.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        IxModule,
+        FormsModule,
+    ],
+})
+export class DateDropdownWrapperComponent implements ControlValueAccessor {
+  @ViewChild(IxDateDropdown) private ixDateDropdown: IxDateDropdown;
+
+  value: string;
+  onChange: (value: string) => NonNullable<unknown>;
+  onTouched:  () => NonNullable<unknown>;
```
Comment (KonsumGandalf) [inline libs/cases/frontend/view/src/lib/components/create-case/date-dropdown-accessor.ts]: Add docs for what it is. Why do you use the format yyyy-MM-dd ? Its uncommon for germany.

**Review by KonsumGandalf — DISMISSED** [review]
Some improvements but looks good to me

File: libs/cases/frontend/view/src/lib/components/create-case/create-case.component.ts (hunk 1 above)
Comment (PatrickSchm1dt) [reply libs/cases/frontend/view/src/lib/components/create-case/create-case.component.ts]: At the beginning I wanted to use the dynamic Forms, but Ix uses template driven Forms: https://ix.siemens.io/docs/controls/validation

**Review by HaruspexSan — APPROVED** [review]
Looked at the details page which caused some merge conflicts, these are all resolved now. As it was my issue for the sprint. Looks good - keep the good work up.

**Review by PatrickSchm1dt — APPROVED** [review]

**Pull request merged**